README.md

```diff
@@ -243,6 +243,13 @@ URL with servo).
 
 ### Commandline Arguments
 
+Usage:
+
+```sh
+./servo [url] [command] # if you run with nightly build
```

Comment:
nit: maybe `[arguments]`? instead of `[command]`?

Reply:
sure. I will change